### PR TITLE
Added signup and login page

### PR DESCRIPTION
**Pull request by shahbazibnafzal**
1. Added Signup page
2. Added Login page

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            </section>
+
+        </main>
+
+        <footer class="dark-bg padder-vh-xl light-text">
+            <div class="footer-grid">
+                <div class="footer-column footer-company-column">
+                    <div class="footer-logo">
+                        <div class="brand-logo">
+                            <img class="img-responsive" src="/images/bb-dark.png"/>
```
Comment (Maianki) [inline login.html]: Add ```alt``` attribute to image for accessibility purpose

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+<body>
+
+    <div class="auth-grid-container">
+
+        <header>
+            <div class="navbar-section">
+                <nav class="navbar light-bg shadow-dark">
+                    <a href="/index.html" class="navbar-brand">
+                        <div class="brand-logo">
+                            <img class="img-responsive" src="/images/bb-light.png"/>
```
Comment (Maianki) [inline signup.html]: Here also you can add ```alt```

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                </form>
+            </section>
+        </main>
+
+        <footer class="dark-bg padder-vh-xl light-text">
+            <div class="footer-grid">
+                <div class="footer-column footer-company-column">
+                    <div class="footer-logo">
+                        <div class="brand-logo">
+                            <img class="img-responsive" src="/images/bb-dark.png"/>
```
Comment (Maianki) [inline signup.html]: Add ```alt``` attribute here too.

**Review by Maianki — APPROVED** [review]
Great job. Just add alt attribute to all your images.

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, just a suggestion the UI, login and Signup page are quite wide,
 it would look better if they are less wider..
also in password we need to add a password check functionality as per the LMS guide (I also missed it initially, then implemented later, do have a look in LMS).

**Review by BalajiHarinath — APPROVED** [review]
The code looks good. Make the changes which Ankit suggested for img tag.
One suggestion on UI :-
I think you can reduce the width of the sign up/login form.
Have a look at P0 requirements for sign up page it says you should have first name and last name input fields as well along with password and confirm password input fields.

File: signup.html (hunk 3 above)
Comment (shahbazibnafzal) [reply signup.html]: Added, Thanks

File: login.html (hunk 1 above)
Comment (shahbazibnafzal) [reply login.html]: Added, Thanks

File: signup.html (hunk 2 above)
Comment (shahbazibnafzal) [reply signup.html]: Added, Thanks

**Comment by shahbazibnafzal** [comment]
> The code looks good. Make the changes which Ankit suggested for img tag. One suggestion on UI :- I think you can reduce the width of the sign up/login form. Have a look at P0 requirements for sign up page it says you should have first name and last name input fields as well along with password and confirm password input fields.

Fixed, Thanks

**Comment by shahbazibnafzal** [comment]
> The code looks good. Make the changes which Ankit suggested for img tag. One suggestion on UI :- I think you can reduce the width of the sign up/login form. Have a look at P0 requirements for sign up page it says you should have first name and last name input fields as well along with password and confirm password input fields.

Worked on the suggestions, 
For now I think we have to work for styling only. Added the style related changes. Thanks :)

**Comment by BalajiHarinath** [comment]
> > The code looks good. Make the changes which Ankit suggested for img tag. One suggestion on UI :- I think you can reduce the width of the sign up/login form. Have a look at P0 requirements for sign up page it says you should have first name and last name input fields as well along with password and confirm password input fields.
> 
> Worked on the suggestions, For now I think we have to work for styling only. Added the style related changes. Thanks :)

Looks good now. You can merge it.

**Comment by shahbazibnafzal** [comment]
Thanks @BalajiHarinath , @Anurag021 , @Maianki  for all the feedback and approvals. Merging it to dev.

**Pull request merged**